packages/swig.rb

```diff
@@ -3,34 +3,35 @@
 class Swig < Package
   description 'Simplified Wrapper and Interface Generator'
   homepage 'http://www.swig.org'
-  version '4.0.1'
+  version '4.0.2'
   license 'GPL-3, BSD and BSD-2'
   compatibility 'all'
-  source_url 'http://prdownloads.sourceforge.net/swig/swig-4.0.1.tar.gz'
-  source_sha256 '7a00b4d0d53ad97a14316135e2d702091cd5f193bb58bcfcd8bc59d41e7887a9'
+  source_url 'http://prdownloads.sourceforge.net/swig/swig-4.0.2.tar.gz'
```

Comment:
Change to `https://downloads.sourceforge.net/project/swig/swig/swig-4.0.2/swig-4.0.2.tar.gz`?  IOW, use https.

Reply:
Done.